pages/_app.js

```diff
@@ -58,7 +58,7 @@ function getComponents(page = {}) {
 
 const requirePage = require.context('.', true, /\.(js|md)x?$/)
 
-export default class MyApp extends App {
+export class BlueprintsApp extends App {
```

Comment:
I don't think we need to export here since we're also exporting at the end of the file